monitoring/package.json

```diff
@@ -14,8 +14,8 @@
   },
   "dependencies": {
     "@keep-network/tbtc-v2-mainnet": "npm:@keep-network/tbtc-v2@mainnet",
-    "@keep-network/tbtc-v2-testnet": "npm:@keep-network/tbtc-v2@goerli",
-    "@keep-network/tbtc-v2.ts": "development",
+    "@keep-network/tbtc-v2-testnet": "npm:@keep-network/tbtc-v2@sepolia",
+    "@keep-network/tbtc-v2.ts": "1.4.0-dev.1",
```

Comment:
Why do we need this particular package?

Reply:
We can't leave `development` as it will fetch an incompatible `2.4.0-dev.X` version. I changed to `1.4.0-dev.1` as this is the version used in `yarn.lock` and we are sure it is compatible.